### PR TITLE
Include location in the response when querying HGV annual tests

### PR DESCRIPTION
**Pull request by mateuszc-kainos**
Add location to annual test response
Introduce new v7 mapper
Make v7 mapper the default one

### CODE REVIEW

File: api/src/main/java/uk/gov/dvsa/mot/app/TradeServiceRequestHandler.java [hunk 1]
```diff
@@ -435,9 +435,8 @@ private String parseVersionNumber(ContainerRequestContext requestContext) {
     }
 
     /**
-     * This method accepts the query string of VRMs (separated by commas) and parses them into
-     * a trimmed set.
-     *
+     * This method accepts the query string of VRMs (separated by commas) and parses them into a trimmed set.
+     * <p>
```
Comment (aaronddvsa) [inline api/src/main/java/uk/gov/dvsa/mot/app/TradeServiceRequestHandler.java:439]: Should this tag be here?

Comment (mateuszc-kainos) [reply api/src/main/java/uk/gov/dvsa/mot/app/TradeServiceRequestHandler.java:439]: This tag was introduced by IntelliJ autoformatting using DVSA code style. I noticed that this tag is present in some other classes, e.g. TradeServiceRequestHandlerTest, line 200, that's why I didn't remove it. I'm happy to do so if you prefer.

File: api/src/test/unit/java/uk/gov/dvsa/mot/app/TradeServiceRequestHandlerTest.java [hunk 2]
```diff
@@ -623,6 +624,31 @@ public void getTradeAnnualTests_Registrations_DefiningOneRegistrationReturnsOneV
...
+
+        when(requestContext.getHeaders()).thenReturn(headers);
+        when(tradeAnnualTestsReadService.getAnnualTests(any())).thenReturn(Arrays.asList(vehicle, vehicle1));
+
+        createHandlerAndGetTradeAnnualTests(request);
+
+        verify(searchVehicleResponseMapperFactory).getMapper(argumentCaptor.capture());
+        assertNull(argumentCaptor.getValue());
+    }
+
```
Comment (aaronddvsa) [inline api/src/test/unit/java/uk/gov/dvsa/mot/app/TradeServiceRequestHandlerTest.java:708]: Do we need a test to exercise the handler when it's called with v7 request? (That may already be happening but I can't see where)

Comment (mateuszc-kainos) [reply api/src/test/unit/java/uk/gov/dvsa/mot/app/TradeServiceRequestHandlerTest.java:708]: Ok. I added tests checking if proper mapper version was invoked.

**Review by aaronddvsa — APPROVED** [review]

**Review by DanielKChristou — APPROVED** [review]

**Pull request merged**